config/app.yml

```diff
@@ -59,6 +59,9 @@ defaults: &defaults
     economics:
       title: Principles of Economics
       sf: Economics
+    introduction_to_business:
```

Comment:
Since this same info is in the CMS, it seems like we should be pulling it from there. I'll create a card.